content/docs/coldchain/equipment.md

```diff
@@ -0,0 +1,210 @@
+
+![Equipment filters](/docs/coldchain/images/equipment_filter.png)
+
+#### Delete Equipment
+
+Assets can be selected and deleted using the toolbar action:
+
+<p align="center">
+    <img src="/docs/coldchain/images/delete_rows.png" width="250" height="375">
+</p>
+
+### Importing And Exporting
+
+#### Import
+
+Cold chain assets can be imported from a csv file using the `Import` button
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_button.png" width="550" height="280">
+</p>
+
+This will open an import modal.
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_modal.png" width="750" height="420">
+</p>
+
+An example template comma-separated-value (csv) is available for download here:
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_template_download.png" width="750" height="420">
+</p>
+
+When importing assets from a store, they will be assigned to that store.
+When importing assets from the [central server](/docs/administration/synchronisation/), an additional field of `store` will be required which will assign assets to their respective stores.
+
+Information will need to be entered in the format provided by the csv template in order for Open mSupply to be able to process and upload this data.
+
+A csv file can be uploaded once it has been created in the example format.
+
+<p align="center">
+    <img src="/docs/coldchain/images/equipment_import_upload.png" width="750" height="420">
+</p>
+
+#### Export
+
+Asset can be exported as a csv using the `Export` button
+
+<p align="center">
+    <img src="/docs/distribution/images/export.png" width="150" height="67">
+</p>
+
+### Create new asset
+
+Cold chain assets can be created manually with the `New Asset` button
+
+<p align="center">
+    <img src="/docs/coldchain/images/create_asset_button.png" width="150" height="55">
+</p>
+
+Clicking this button will open the `Create Asset` modal
+
+With this modal you can assign:
+
+- the Asset Category
+- the asset catalogue item the asset belongs to
+- the Asset number
+- notes about the asset
+
+If the asset you are creating is not part of the `WHO PQS catalogue`, you can use the toggle to create an asset which does not belong to the catalogue.
+
+<p align="center">
+    <img src="/docs/coldchain/images/create_asset_modal.png" width="650" height="420">
+</p>
+
+### Managing cold chain assets
+
+Clicking on a row in the list will show the details of that asset.
+
+![Asset detail view](/docs/coldchain/images/asset_detail_view.png)
+
+There are 4 tabs available to manage the asset:
+
+![Detail view tabs](/docs/coldchain/images/asset_detail_view_tabs.png)
+
+#### Summary tab
+
+The `Summary` tab can
+
+- edit the serial number
+- assign an installation date
+- assign a replacement date
+- assign locations to the the asset
+
+Multiple locations can be assigned to one asset.
```

Comment:
```suggestion
<div class="tip">Multiple locations can be assigned to one asset</div>
```

will give you this:

<img width="534" alt="image" src="https://github.com/msupply-foundation/msupply_docs/assets/9192912/20c4ea5f-ee21-4a24-9ff7-67c65b4186e3">